generic/piranha-tree-sitter/src/test-resources/java/configurations/rules.toml

```diff
@@ -0,0 +1,453 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+# 
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+# 
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file contains rules to the specific feature flag API.
+
+#
+# For @stale_flag_name = STALE_FLAG and @treated = true
+# Before 
+#  exp.isToggleEnabled(Experiment.STALE_FLAG)
+# After 
+#  true
+#
+[[rules]]
+name = "replace_isToggleEnabled_with_boolean_literal"
+query = """((
+	(method_invocation 
+    	name : (_) @name
+        arguments: ((argument_list 
+        				([
+                          (field_access field: (_)@argument)
+                          (_) @argument
+                         ])) )
+        	
+    ) @method_invocation
+)
+(#eq? @name "isToggleEnabled")
+(#eq? @argument "@stale_flag_name")
+)"""
+replace_node = "method_invocation"
+replace = "@treated"
+groups = [ "replace_expression_with_boolean_literal"]
+holes = ["treated", "stale_flag_name"]
```

Comment:
assume the API can also be a hole.

Reply:
Yes, API could be a hole too. We can do that as a follow up.